src/tests/multihost/sssd/testlib/common/qe_class.py

```diff
@@ -191,6 +191,16 @@ def dnf_uninstall(self, package):
                                raiseonerr=False)
         return cmd.returncode
 
+    def detect_files_provider(self):
+        """
+        Detect the SSSD's files provider feature 
+        :param: None
+        :return str: Index 1's element from the command
+        :Exception: None
+        """
+        check = f'ls "/usr/lib64/sssd/libsss_files.so" && echo "files-provider"'
+        cmd = self.run_command(check, raiseonerr=False)
+        return cmd.stdout_text.split('\n')[1]
```

Comment:
Just check if "/usr/lib64/sssd/libsss_files.so" exists and return true or false.

Reply:
yes, updated, please check again